loading-code/typescript-default-extenstion/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Basic Typescript SystemJS Example</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <script type="systemjs-importmap">
+    {
+      "imports": {
+        "lodash": "https://cdn.jsdelivr.net/npm/lodash@4.17.15/lodash.min.js"
+      }
+    }
+  </script>
+  <script src="https://cdn.jsdelivr.net/npm/systemjs/dist/system.js"></script>
+  <script>System.import('./dist/systemjs-hooks/fetch.js')</script>
```

Comment:
It's preferred to load systemjs extras / hooks via normal script tag instead of via System.import. The reason is that once the first import occurs, certain hooks won't work right (for example, `prepareImport()`). In this case, it doesn't seem to break anything, but I think it would be better to show the best practice here:

```suggestion
  <script src="/dist/systemjs-hooks/fetch.js"></script>
```